src/microbe_stage/gui/PatchDetailsPanel.cs

```diff
@@ -820,6 +843,16 @@ private void MigrationAmountSliderChanged(double value)
         UpdateMigrationStatusText();
     }
 
+    private float CalculateOtherCompoundAmount()
```

Comment:
```suggestion
    private float CalculateUnaccountedForCompoundPercentage()
```

I think this kind of name would be more clear.